monkestation/code/game/objects/items/plasma_license.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/card/plasma_license
-	name = "License to Plasmaflood"
-	desc = "A charred contract letting the holder plasmaflood the halls. Not offically recognized by Nanotrasen."
+	name = "License to Murderbone"
+	desc = "A charred contract letting the holder kill everyone they want. Not offically recognized by Nanotrasen."
```

Comment:
For some reason this reminds me of Meet The Sniper.

Because of that, and because "kill everyone they meet" sounds more badass to me:
```suggestion
	desc = "A charred contract letting the holder kill everyone they meet. Not offically recognized by Nanotrasen."
```

(FWIW, you don't have to apply this. Just figured I'd suggest it.)

Reply:
i like this idea